releasenotes/notes/0.12/tensor_network_gpu-e8eb3e40be3c35f7.yaml

```
@@ -0,0 +1,24 @@
+---
+features:
+  - |
+    Added a new simulation method based on NVIDIA's `cuTensorNet <https://docs.nvidia.com/cuda/cuquantum/cutensornet/index.html>`__
+    APIs of cuQuantum SDK. This provides a GPU accelerated general tensor
+    network simulator that can simulate any quantum circuit, by internally
+    translating the circuit into a tensor network to perform the simulation.
+    To use this simulation method, set ``method=tensor_network`` and
+    ``device=GPU`` when initializing an :class:`~.AerSimulator` object.
+    For example::
+
+        from qiskit_aer import AerSimulator
+
+        tensor_net_sim = AerSimulator(method=tensor_network, device=GPU)
+
+    This method supports both statevector and density matrix simulations.
+    Noise simulation can also be done with a density matrix single shot
+    simulation ith density matrix are is not :class:`~.SaveStatevector` operations
+    in the circuit.
```

i'm not sure what this sentence is trying to say, but something goes funny in the middle of it

---

Lol, I clearly shut off my brain while editing this. Let me fix it, it's suppsed to be:

```suggestion
    Noise simulation can also be done with a density matrix single shot
    simulation if there are not any :class:`~.SaveStatevector` operations
    in the circuit.
```